services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/tormaystarkastelu/TormaystarkasteluCalculator.kt

```diff
@@ -105,7 +105,7 @@ object TormaystarkasteluCalculator {
             tormaystarkasteluTulos.pyorailyIndeksi = 1.0f
         } else {
             tormaystarkasteluTulos.pyorailyIndeksi =
-                calculationRule.tulos * calculationRule.painotus // TODO: 1 decimal precision?
+                calculationRule.tulos * calculationRule.painotus
```

Comment:
conflict with Teemus version of this change? (Teemu added the rounding)  (Same with other similar rows)